Content.Server/Radio/EntitySystems/RadioDeviceSystem.cs

```diff
@@ -206,6 +207,15 @@ private void OnReceiveRadio(EntityUid uid, RadioSpeakerComponent component, ref
         _chat.TrySendInGameICMessage(uid, args.Message, InGameICChatType.Whisper, ChatTransmitRange.GhostRangeLimit, nameOverride: name, checkRadioPrefix: false);
     }
 
+    private void OnIntercomInit(Entity<IntercomComponent> ent, ref MapInitEvent args)
+    {
+        foreach (var id in ent.Comp.SupportedChannels)
+        {
+            var channel = _protoMan.Index<RadioChannelPrototype>(id);
+            ent.Comp.SupportedFrequencies.Add(channel.Frequency);
+        }
+    }
```

Comment:
i dont really like this pattern of having to separately store frequencies and bookkeep it to make sure all the loaded prototypes get their frequencies added. an alternative is to maybe make frequency just into a string and have it all in one list. try to index it as a radio channel first and if it doesnt work assume its a frequency? maybe janky